docs/css_types/border.md

```diff
@@ -0,0 +1,59 @@
+# &lt;border&gt;
+
+The `<border>` CSS type represents a border style.
+
+## Syntax
+
+--8<-- "docs/snippets/type_syntax/border.md"
+
+## Border command
+
+The `textual` CLI has a subcommand which will let you explore the various border types interactively, when applied to the CSS rule [`border`](../styles/border.md):
+
+```
+textual borders
+```
+
+## Examples
+
+### CSS
+
+```sass
+* {
+    rule: ascii   /* A border with plus, hyphen, and vertical bar characters. */
```

Comment:
I guess the `rule` is meant as a placeholder?

This doesn't really serve as an example, since it is invalid CSS.

I think what we need from the examples section is just a few examples of use so the reader can orient themselves. Doesn't have to be an exhaustive list, which is covered nicely in the Syntax section.

Suggest small fragments. For the border something like this:

```sass
#container {
    border: heavy red;
}

#heading {
    border-bottom: solid blue;
}
```

Something similar for the Python section.

Reply:
Fixed in 81ddae7